blob/service_test.go

```diff
@@ -1,5 +1,3 @@
-//go:build linux
```

Comment:
It's kinda surprising that this was anywhere helpful. Do you have any clue?

Reply:
Yes, it was added by @ramin because macOs builds were flaky. We agreed to remove it once I fix this issue.